packages/lesswrong/server/callbacks/postCallbacks.ts

```diff
@@ -178,3 +178,10 @@ export async function shouldNewDocumentTriggerReview (document) {
   return document
 }
 addCallback("posts.new.after", shouldNewDocumentTriggerReview);
+
+async function shouldUpdatedPostTriggerReview (newPost, oldPost) {
+  if (!newPost.draft && oldPost.draft) {
+    shouldNewDocumentTriggerReview(newPost)
+  }
+}
+addCallback("posts.edit.async", shouldUpdatedPostTriggerReview);
```

Comment:
Not new in this PR, but nitpick: the function name `shouldUpdatedPostTriggerReview` is a question, which means it should be a pure function returning boolean, but is actually a function with side-effects. A better name would be `newDocumentMaybeTriggerReview` (which is a valid name for a function with side-effects because it's an imperative sentence).